### PR TITLE
ace_spectator documentation

### PR DESCRIPTION
**Pull request by SilentSpike**
Work in progress documentation on the new ace_spectator feature. Includes:
- Configuration details
- Shortcut details
- Interface notes
- Module description

### CODE REVIEW

**Comment by SilentSpike** [comment]
If a second pair of eyes wants to give this the once over that'd be perfect :smile:

File: documentation/feature/spectator.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+parent: wiki
+---
+
+## 1. Overview
+
+The ACE3 spectator system is designed to act as a flexible and easy to configure framework. Most scenarios can be set up as desired using only the settings provided, however public functions are available for finer control of these configurable aspects.
+
+### 1.1 Spectator System
+
+The current itteration of the ACE3 spectator system only officially supports scenarios using respawn type 3 (or "BASE"). However there's nothing to stop its use alongside anything else, just be aware that it might not function entirely as expected.
```
Comment (bux578) [inline documentation/feature/spectator.md]: iteration

File: documentation/feature/spectator.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+By default, the ACE3 spectator system does nothing - meaning existing missions will behave exactly as before. The setting `ace_spectator_onDeath` can be used to automatically put players into spectator mode each time they die.
+
+For scenario designers who seek a more advanced setup (such as multiple lives or wave respawning) the function `ace_spectator_fnc_setSpectator` is provided to transition players to/from spectator mode as desired:
```
Comment (bux578) [inline documentation/feature/spectator.md]: we're calling them "mission makers" everywhere atm.
I like "scenario designers" better, though

File: documentation/feature/spectator.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+ * Available vision modes <ARRAY>
+ *
+ * Example:
+ * [[0], [1,2]] call ace_spectator_fnc_updateVisionModes
+```
+
+### 1.6 Camera Attributes
+
+The spectator camera has 8 manipulatable attributes:
+- Camera mode: The camera view
```
Comment (bux578) [inline documentation/feature/spectator.md]: a little bit of text formatting would be awesome here. Maybe have the options "bold" and the description "normal". Or options "normal" and description "italic".

**Comment by SilentSpike** [comment]
:+1: Thanks, all changes made

**Comment by alganthe** [comment]
:+1:

**Comment by SilentSpike** [comment]
Merging :smile:

**Pull request merged**